### DISCUSSION TITLE
Why not add topics?

### DISCUSSION THREAD

**Discussion by C-BJ** [carbon-language/carbon-lang#1460] [General]
Adding topic can let more people interested in this project know about this project
`carbon-lang` `c++` `language` `programming-language` `compiler`

**Comment by masters3d** [comment]
There are some topics now

**Reply by C-BJ** [reply]
> There are some topics now

I meant the topics of the project rather than the topics discussed
btw: Do the topics of discussions need q&a

**Reply by jonmeow** [reply]
Sorry, I actually thought I'd commented here when I added the discussion categories -- but I see I also misunderstood! I've added repository topics as you've suggested, thank you!

We didn't have many categories because we mainly use Discord and Google Docs. But this did make me realize we should probably add a few more and see where things go -- thus Toolchain, Explorer, and Language design. Getting started is the Q&A topic for now, to see how it's used.

**Reply by C-BJ** [reply]
> Sorry, I actually thought I'd commented here when I added the discussion categories -- but I see I also misunderstood! I've added repository topics as you've suggested, thank you!
> 
> We didn't have many categories because we mainly use Discord and Google Docs. But this did make me realize we should probably add a few more and see where things go -- thus Toolchain, Explorer, and Language design. Getting started is the Q&A topic for now, to see how it's used.

Great, But for the Q&A provided by GitHub by default, There is a recognizable icon and the correct answer can be mark

**Reply by jonmeow** [reply]
I'd just missed that icons could be set. I looked at another repo (https://github.com/prettier/prettier/discussions) and used similar icons for common categories.
